tests/infrastructure/prometheus.go

```diff
@@ -149,15 +149,10 @@ var _ = DescribeInfra("[rfe_id:3187][crit:medium][vendor:cnv-qe@redhat.com][leve
 		}
 		Expect(promPort).ToNot(BeEquivalentTo(0), "could not get Prometheus port from endpoint")
 
-		// We need a token from a service account that can view all namespaces in the cluster
```

Comment:
Suggestion: Leave a comment that the SA needs to have access to the Prometheus subresource `api`